src/main/java/mate/academy/model/User.java

```diff
@@ -0,0 +1,65 @@
+package mate.academy.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import java.util.Arrays;
+
+@Entity
+@Table(name = "users")
+public class User {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String email;
+    private String password;
+    private byte[] salt;
+
+    public Long getId() {
+        return id;
+    }
+
+    public User setId(Long id) {
```

Comment:
SETTERS shouldn't return values, change it in all setters!

```suggestion
    public void setId(Long id) {
```